share/completions/rc-status.fish

```diff
@@ -1,7 +1,10 @@
+set -l levels_path /etc/runlevels/*
```

Comment:
Please don't do this *here*.

That means it will be done when the completions are *loaded*.

Do it all inside the argument string:

```fish
set -l levels '(set -l p /etc/runlevels/*; string replace /etc/runlevels "" $p)'
```

(and yes, the `set` is necessary because `string` will error out if the glob doesn't match)